### PR TITLE
Release GE plugin 3.5

### PR DESCRIPTION
**Pull request by felipefzdz**

### CODE REVIEW

**Review by wolfs — APPROVED** [review]
LGTM! Let's not merge this to `release`, only if you are sure there will be a 6.7.1. And even with a 6.7.1 I wouldn't be confident to upgrade the GE plugin version there.

**Comment by felipefzdz** [comment]
Thanks @wolfs! I've changed the base branch to `master`. Feel free to merge it if that's the branch that you were thinking on.

**Pull request merged**
